src/network/netdev/netdev.c

```diff
@@ -714,10 +714,6 @@ int netdev_load_one(Manager *manager, const char *filename) {
                 return 0;
         }
 
-        r = fseek(file, 0, SEEK_SET);
```

Comment:
Is `file` used for anything? It looks like creation of `file` could be dropped altogether.

EDIT: oh, it's removed in the next commit. So maybe combine those in one patch?